suprb/optimizer/solution/ga/crossover.py

```diff
@@ -53,3 +53,33 @@ def _crossover(self, A: Solution, B: Solution, random_state: RandomState) -> Sol
         genome[~indices] = B.genome[~indices]
 
         return A.clone(genome=genome)
+
+
+class SagaCrossover(BaseComponent):
+    """Performs crossover and mutation of Parameters, then calls resulting crossover function"""
```

Comment:
Parameters is a bit too unspecific.

Do not call it SagaCrossover but SACrossover or SelfAdaptiveCrossover. Given that this is not universal, it would make sense to mention the relevant papers here

Reply:
Changed